packages/paste-website/src/pages/components/blockquote/index.mdx

```diff
@@ -0,0 +1,192 @@
+[Callout](/components/callout) | Use a Callout to highlight any other important information. | <Callout variant='neutral'><CalloutHeading as="h2">Heads up!</CalloutHeading><CalloutText>This is some information you need to know.</CalloutText></Callout>
+
+## Examples
+
+### Blockquote
+
+Use a default Blockquote when you want to highlight a quote from an external source, but there is no specific need for URL and the context is clear.
+
+<LivePreview
+  scope={{Blockquote, BlockquoteContent, BlockquoteCitation}}
+  language="jsx"
+>
+  {`<Blockquote url="#">
+      <BlockquoteContent>
+        With AI-driven products, the design process is no longer just about aesthetics. It’s about designing for the
+        human experience as a whole.
+      </BlockquoteContent>
+      <BlockquoteCitation author="Google" source="People + AI Guidebook" />
+    </Blockquote>`}
+</LivePreview>
+
+### Blockquote without URL
+
+Use Blockquote without URL when you have the actual source of the quote but are unable to provide a URL to redirect the user. 
+
+<LivePreview
+  scope={{Blockquote, BlockquoteContent, BlockquoteCitation}}
+  language="jsx"
+>
+  {`<Blockquote>
+      <BlockquoteContent>
+        If I must die <br/>
+        let it bring hope <br/>
+        let it be a tale
+      </BlockquoteContent>
+      <BlockquoteCitation author="Rafaat Alareer" source="If I Must Die" />
+    </Blockquote>`}
+</LivePreview>
+
+### Blockquote with author only
+
+Use a Blockquote with only author attribution when you want to highlight a quote from an external source, but there is no specific need for source or the source is unavailable.
+
+Example use cases:
+- When quoting a well-known individual, where the author’s name provides enough context for the audience.
+- When quoting from a source that is hard to trace or not directly accessible (e.g., a verbal interview or a widely repeated saying without a specific source)
+
+<LivePreview
+  scope={{Blockquote, BlockquoteContent, BlockquoteCitation}}
+  language="jsx"
+>
+  {`<Blockquote>
+      <BlockquoteContent>
+        When you light a candle, you also cast a shadow.
+      </BlockquoteContent>
+      <BlockquoteCitation author="Ursula K. Le Guin" />
+    </Blockquote>`}
+</LivePreview>
+
+## Composition notes
+
+- Whenever possible, include a URL to the original source to allow users to access additional context or details
+- Always provide context before and after the Blockquote to connect the information to the surrounding content
+- Use **capital case** for author’s name. For example: Jane Doe
+- Use **sentence case**, with only the first word and proper nouns capitalized for the source and its URL
+- Blockquote should maintain a vertical spacing of `$space-70` from its surrounding content
```

Comment:
since we have added margin in the component itself, we don't need this composition note anymore